pkg/k8shandler/desirednodestate.go

```diff
@@ -22,6 +22,7 @@ const (
 	elasticsearchCertsPath    = "/etc/openshift/elasticsearch/secret"
 	elasticsearchConfigPath   = "/usr/share/java/elasticsearch/config"
 	elasticsearchDefaultImage = "docker.io/openshift/origin-logging-elasticsearch5"
+	proxyDefaultImage         = "quay.io/brancz/kube-rbac-proxy:v0.4.0"
```

Comment:
do we know if there is a way we can get this into a different repo?

Reply:
What's the difference between this proxy and the oauth-proxy?

Reply:
"quay.io/openshift/kube-rbac-proxy"